code/modules/sanity/sanity_mob.dm

```diff
@@ -256,11 +281,11 @@
 			switch(effect_prob)
 				if(1 to 25)
 					M.stats.addTempStat(STAT_COG, 5, INFINITY, "Fate Nihilist")
-				if(26 to 50)
+				if(25 to 50)
 					M.stats.removeTempStat(STAT_COG, "Fate Nihilist")
-				if(51 to 75)
+				if(50 to 75)
 					penalty *= -1
-				if(76 to 100)
+				if(75 to 100)
```

Comment:
These lines looks suspicious

Reply:
uhm, this is an error that I introduced when coding this, at that time I didn't really know how the switch worked.